cmake/Modules/FindSystemBLAS.cmake

```diff
@@ -27,23 +26,42 @@ set(SystemBLAS_FOUND FALSE)
 
 include(FindPackageHandleStandardArgs)
 
-find_library(OPENBLAS_LIBRARIES NAMES openblas libopenblas)
-find_path(OPENBLAS_INCLUDE_DIRS openblas_config.h)
-if(OPENBLAS_LIBRARIES AND OPENBLAS_INCLUDE_DIRS)
+# Let user possibility to set blas path to similar libraries like lapack
+if(BLAS_LIBRARIES AND BLAS_INCLUDE_DIRS)
   find_package(Threads REQUIRED)
   add_library(blas::blas UNKNOWN IMPORTED)
   set_target_properties(blas::blas PROPERTIES
-    INTERFACE_INCLUDE_DIRECTORIES "${OPENBLAS_INCLUDE_DIRS}"
+    INTERFACE_INCLUDE_DIRECTORIES "${BLAS_INCLUDE_DIRS}"
     INTERFACE_LINK_LIBRARIES Threads::Threads
-    IMPORTED_LOCATION "${OPENBLAS_LIBRARIES}"
+    IMPORTED_LOCATION "${BLAS_LIBRARIES}"
   )
-  set(OPENBLAS_FOUND TRUE)
-  set(SystemBLAS_LIBRARIES OPENBLAS_LIBRARIES)
+  set(BLAS_FOUND TRUE)
+  set(SystemBLAS_LIBRARIES USER_DEFINED)
 else()
+  set(BLA_VENDOR OpenBLAS)
   find_package(BLAS QUIET)
+  if(BLAS_FOUND)
+    set(OPENBLAS_FOUND TRUE)
+    set(SystemBLAS_LIBRARIES OPENBLAS_LIBRARIES)
```

Comment:
it does but it is not what is set here. To set SystemBLAS_LIBRARIES to the value set by findBLAS it should be 
```suggestion
    set(SystemBLAS_LIBRARIES ${BLAS_LIBRARIES})
```
That will print on screen the location of library. This is not what it did before, even though it can be more useful.